scripts/README.md

```diff
@@ -1,39 +1,52 @@
 # Installation Scripts Guide
 
-This document provides instructions for using bash scripts to install Ontario CKAN 2.9 and Solr 8.11.2 on individual Ubuntu 20.04 virtual machines.
+Bash scripts to install CKAN 2.9 compliant Ontario Theme, and Solr 8.11.2 on a separate Ubuntu 20.04 virtual machine. The objective of the scripts is to create a local dev-test environment.
 
-## Solr Installation
+During the course of the script execution, the code repository will be cloned into `/usr/lib/ckan/default/src`for installation. This directory and it being cloned are independent of the code used for installation purposes. This is the directory where the code changes need to be made for dev work.
 
-### Bash Script Information
-- Filename: `setup_solr.sh`
-- Version: Solr 8.11.2
+### Common Steps
 
-This script requires the `managed-schema` config file, customized for Ontario CKAN. This config file is stored in the `config/solr` directory of this repository and copied into the Solr config folder (`/opt/solr/server/solr/configsets/_default/`) during the script execution.
+The installation includes some common steps that are aggregated here. They mentioned in the instructions whenever their execution is needed
 
-Take note, this script handles Solr installation, but does not rebuild the index. You must perform this step on the machine where CKAN is running.
+1. Export your sudo password as an environment variable named `SUDOPASS`:
 
-### Script Execution Steps
+```bash
+export SUDOPASS='xxxx'
+```
 
-**Step 1:** Export your sudo password as an environment variable, `SUDOPASS`. Example command:
+2. Install git
+
+```bash
+sudo -y apt-get install git
 ```
-export SUDOPASS='your_sudo_password'
+
+3. Clone this repository
+
+```bash
+git clone https://github.com/ongov/ckanext-ontario_theme.git
 ```
 
-**Step 2:** Install Git, clone the repository, fetch the `ckan_2.9_upgrade` branch, switch to the scripts directory, and execute the script. Follow the commands below:
+4. change to the scripts directory
 
 ```bash
-# Update packages
-echo $SUDOPASS | sudo -S -k apt-get update
+cd ckanext-ontario_theme\scripts
+```
+
+## Solr Installation
 
-# Install Git
-{ echo $SUDOPASS; echo 'Y'; } | sudo -S -k apt-get install git
```

Comment:
It would be best to keep this rather than the current line 20
![image](https://github.com/ongov/ckanext-ontario_theme/assets/60950452/9e5b39e3-4c6d-4023-96ca-f1f35891abcb)

Reply:
done